config/metadata/configMeta.yaml

```diff
@@ -1330,6 +1330,40 @@ groups:
           activity, then it pings the client to see if the transport is still
           alive. "0s" sets duration to 20 seconds.
 
+      - name: MaxSendMsgSize
+        type: memorysize
+        valuetype: memorysize
+        default: 6MiB
+        reload: false
+        firstversion: v2.2
```

Comment:
Oh, this is the first time I noticed we have this field for a config. Does this mean all new configs since last release should have this set?

Reply:
Yes, technically. It's not currently used for anything, but it could be helpful both for validation and documentation. It might be worth an effort to go back to add those notations to new things that came along in 2.0 and later.